src/common/components/user-nav/index.tsx

```diff
@@ -277,7 +279,7 @@ class UserNav extends Component<Props, State> {
                     {unread.toString().length < 3 ? unread : "..."}
                   </span>
                 )}
-                {global.notifications ? bellSvg : bellOffSvg}
+                {ls.get("notifications") || global.notifications ? bellSvg : bellOffSvg}
```

Comment:
Same here, api response be stored in redux and accessed instead of local storage which is sometimes unreliable

Reply:
Resolved.